happi/cli.py

```diff
@@ -1146,6 +1154,60 @@ def edit_config():
         subprocess.run([os.environ.get("EDITOR", "vi"), config_filepath])
 
 
+@config.command()
+@click.option("--overwrite/--no-overwrite", "overwrite", default=False,
+              help="Overwrite existing config.")
+def init(overwrite):
+    """Create configuration file with default options."""
+
+    # find config_filepath
+    try:
+        config_filepath = Path(happi.client.Client.find_config())
+        if not overwrite:
```

Comment:
Would you mind putting this `if` statement in an `else:` block to reduce the scope of the `try/except OSError`?

Reply:
Great idea. Something I should be more in the habit of doing.